src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildWithATargetPlatform.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using FluentAssertions;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.IO;
+using System;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToBuildWithATargetPlatform : SdkTest
+    {
+        public GivenThatWeWantToBuildWithATargetPlatform(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("netcoreapp3.1", ".NETCoreApp", "v3.1", "Windows", "7.0")] // Default values pre-5.0
```

Comment:
Add a test that `net5.0` results in an empty target platform identifier and version (or is that already covered elsewhere?).